enums.h

```diff
@@ -2700,3 +2700,23 @@ enum shrine_type {
 	SHRINE_TAINTED = 25,
 	NUM_SHRINETYPE
 };
+
+enum action_id {
+	ACTION_NONE = -1,
+	ACTION_ATTACK = 9,
+	ACTION_RATTACK = 10,
+	ACTION_SPELL = 12,
+	ACTION_OPERATE = 13,
+	ACTION_DISARM = 14,
+	ACTION_PICKUPITEM = 15,  // put item in hand (inventory screen open)

```

Comment:
To differentiate between `ACTION_PICKUPITEM` and `ACTION_PICKUPAITEM` I would suggest using `ACTION_LIFT_ITEM = 15` to lift item to hand, and `ACTION_LOOT_ITEM` to loot item into inventory.

If `ACTION_PICKUPITEM` is from the PSX, the of course we use it.

Reply:
I just reused the same terms as the commands that trigger those actions (CMD_REQUESTGITEM/CMD_REQUESTAGITEM/CMD_GOTOGETITEM/CMD_GOTOAGETITEM), no idea where they come from.

Reply:
Ah, I see. Good.

The `cmd` enum is defined is in PSX SYM file. From https://github.com/sanctuary/psx/blob/master/easy-as-pie/types.h#L1371

```c
	CMD_REQUESTGITEM      = 39,
	CMD_REQUESTAGITEM     = 40,
	CMD_GOTOGETITEM       = 41,
	CMD_GOTOAGETITEM      = 42,
```